### ISSUE TITLE
Can't multiple select to add to collection

### ISSUE THREAD

**Issue by D4rkSl4ve** [MediaBrowser/Emby#3533]
Currently using  `Version 4.1.0.15 beta` for Netgear NAS.
I had to create couple of Movie Collections and I was not able to multiple select movies with different names to add to a collection.  For example, creating a `Bourne Collection` is fairly easy as simply searching for Bourne, all movies come.  But wanting to create a `James Bond Collection`, well, would have to search for each one as only couple have the 'gold' name.  I wasn't able to multiple-select.  I would select, say For Your Eyes Only, and search for Spectre, and even though the movie would show up, I wasn't able to tag/mark the movie as before; where it would continue adding movies with the 4x, 5x, etc.. .and then select the ... followed by the Add to Collection, either New or existing...   just an issue.

**Comment by LukePulverenti** [comment]
Hi, can you describe in more detail what you mean? if you can't get them onto the same screen together, then you'd just have to add them one at a time.

**Comment by D4rkSl4ve** [comment]
> Hi, can you describe in more detail what you mean? if you can't get them onto the same screen together, then you'd just have to add them one at a time.

I could remember when I created my 'Marvel Cinematic Universe' collection I was able to search:  thor, and selected/tagged the movies, then searched captain, select/tag, then avengers, etc... and once I had all `x 20` on the top left of the screen, then I clicked the `...` on the top/right, and `Add to collection`

Now on 4.1.0.15 beta that is not working, and wasn't working on .14 either.  Before that I had 4.0.1.0 it was working.  I went from `emby-server-netgear_4.0.1.0_amd64` to 4.1.0.10, 4.1.0.14, and 4.1.0.15.
